package.json

```diff
@@ -12,20 +12,23 @@
         "typescript": ">=2.4.2"
     },
     "devDependencies": {
-        "@bazel/ibazel": "^0.2.0",
+        "@bazel/ibazel": "git+https://github.com/gregmagolan/bazel-watcher#dist-profiler-2",
         "@types/jasmine": "^2.8.2",
         "@types/node": "7.0.18",
         "@types/source-map": "^0.5.1",
         "clang-format": "1.0.49",
         "concurrently": "^3.5.1",
         "protobufjs": "5.0.0",
         "protractor": "^5.2.0",
+        "puppeteer": "^0.13.0",
+        "tempy": "^0.2.1",
         "tsickle": "0.25.x",
         "tsutils": "2.12.1",
         "typescript": "2.5.x"
     },
     "scripts": {
         "pretest": "webdriver-manager update && bazel build examples/app:all",
-        "test": "concurrently \"bazel run examples/app:devserver\" protractor --kill-others --success first"
+        "test": "concurrently \"bazel run examples/app:devserver\" protractor --kill-others --success first",
+        "benchmark-test": "node devserver-benchmark-runner.js //examples/app:devserver examples/app/app.ts http://localhost:5432 300 10"
```

Comment:
this looks like the right shape solution to me.

How would a user adopt this? copy-paste the runner script? reach into `$(bazel info output_base)/external` to find the copy bazel already fetched? other ideas?

Reply:
Should just be able to run `node_modules/@bazel/typescript/devserver-benchmark-runner.js ...`

Reply:
edited the comment - not that easy...

Reply:
> How would they get the needed runtime deps now that isn't an npm dependency?

Right. The recommended way to get rules_typescript is a git repo now?

> Does this need to be in the typescript layer?

Not necessarily. I suppose it could work with any devserver. Will depend on iBazel with the profiler and the devserver to have livereload. If the user has a iBazel in their path then the script could just run `ibazel` instead of `node_modules/.bin/ibazel`. Can we make that assumption?

Reply:
Maybe the right place for it to live is within the iBazel repo so when you pull in ibazel from npm you get the script and the required deps.

Reply:
Oh. But then there is the case of a non-npm ibazel install. Right.